modules/nw-aws-load-balancer-with-outposts.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+// * networking/aws_load_balancer_operator/understanding-aws-load-balancer-operator.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-aws-load-balancer-with-outposts_{context}"]
+= AWS Load Balancer Operator and AWS Outposts
+
+AWS Load Balancer Operator can work with AWS Outposts. Ingress resources must be annotated with the either regular subnets or outposts subnets (but not both).
+
+.Procedure
+
+* Configure the Ingress resource to use a specified subnet:
++
+.Example Ingress resource configuration
+[source,yaml]
+----
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: echoserver
+  annotations:
+    alb.ingress.kubernetes.io/subnets: <outposts-subnet-id> # <1>
+spec:
+  ingressClassName: alb 
+  rules:
+    - http:
+        paths:
+          - path: /
+            pathType: Exact
+            backend:
+              service:
+                name: echoserver
+                port:
+                  number: 80
+----
+<1> Specifies the subnet to use. To use the Application Load Balancer in an Outpost, specify the Outpost subnet ID.
+
+Application Load Balancer is supported. Classic Load Balancer and Network Load balancer are not, so AWS Load Balancer Operator will not provision them on AWS Outposts subnets.
```

Comment:
```suggestion
Application Load Balancer is supported. Network Load balancer is not, so AWS Load Balancer Operator will not provision it on AWS Outposts subnets.
```

What do you think about moving these phrases at the beginning of the chapter and placing let's say after "AWS Load Balancer Operator can work with AWS Outposts"?